### ISSUE TITLE
Maintenance status

### ISSUE THREAD

**Issue by ghost** [clojure-android/neko#74]
Is this project defunct?

**Comment by MasterMastic** [comment]
I'm curious about that too. Just fwiw, I am using it with ease.

**Comment by ghost** [comment]
@MasterMastic I can't get a grid layout together for the life of me, and I haven't seen anyone post sample code for any projects they used it in since 2015, so I'm debating dropping it.

**Comment by MasterMastic** [comment]
I imagine I could help you. I've been able to use anything I needed from Android so far. Can you tell me what issues you're having?

As for dropping it: Neko so far has been a pleasure for me to work with, however you might want to check out [re-natal](https://github.com/drapanjanas/re-natal). Simply because React Native.

**Comment by zerg000000** [comment]
I think to have someone to maintain this project would be great, Neko is really a pleasure to work with if only Android is invoked. Only the little works would be helpful, e.g. upgrade the dependencies version, accept pull requests. However, re-natal really draws all the attention now...

**Comment by ghost** [comment]
Yes, if this project were more active I would be more inclined to dig into it because "really native". On the other hand, since everything is JIT anyway, I suppose re-natal is more than sufficient
